### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ricardobranco777**
Update README with information on how to use the configuration included in this repo.

### CODE REVIEW

File: gcp/terraform/README.md [hunk 1]
```diff
@@ -1,26 +1,61 @@
...
+
+```
+gsutil mb gs://sles-images
+```
+
+7. Upload the image you want to use with:
+
+`gsutil cp OS-Image-File-for-SLES4SAP-for-GCP.tar.gz gs://sles-images`
+
+Note: [BYOS images](https://www.suse.com/c/on-demand-byos-say-what/) are not supported yet.
```
Comment (alvarocarvajald) [inline gcp/terraform/README.md]: We should not restrict this only to BYOS images. What happens if whoever uses this has their own SUSE Licenses? @hsehic what do you think?

Comment (ricardobranco777) [reply gcp/terraform/README.md]: Added support for BYOS images now.

File: gcp/terraform/README.md [hunk 2]
```diff
@@ -1,26 +1,61 @@
 # Google Cloud Platform deployment with Terraform
 
-The terraform configuration files in this directory can be used to create the infrastructure required to perform the validation of the public cloud images for SLES for SAP Applications in the **Google Cloud Platform**.
+This Terraform configuration deploys SAP HANA in a High-Availability Cluster on SUSE Linux Enterprise Server in the **Google Cloud Platform**.
```
[the remark this answers is not in the document]
Comment (alvarocarvajald) [reply gcp/terraform/README.md]: OS name should be SUSE Linux Enterprise Server for SAP Applications

[the remark this answers is not in the document]
Comment (ricardobranco777) [reply gcp/terraform/README.md]: Done.

**Pull request merged**